app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :posts
+  has_many :likes
```

Comment:
- [ ] In the ERD given, you're requested to make `author_id ` reference key in the `likes` table instead of user_id. Kindly fix :wrench:  this issue and equally check the comment above for more explanations.

Reply:
Hello @FrederickMih, Thanks a lot for the kind suggestions. I am a little confused here. In the ERD the like table has a `userId` not `authorId`. Can you please clarify what you mean.

Reply:
![Screenshot from 2023-06-15 10-25-50](https://github.com/Muhammad0602/Blog-app/assets/63087491/19586dce-bb12-4313-ab15-7fb2e830e4a1)

Reply:
@FrederickMih I am not sure what is happening but my diagram is different.
![image](https://github.com/Muhammad0602/Blog-app/assets/113791476/df409e39-d785-488d-a5ba-d84b08ee2037)

Reply:
and in my tables it is already user_id not author_id. So I think it will affect that as well